test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -112,6 +112,11 @@ var helpers = function helpers() {
         {
             return Object.prototype.toString.call(object);
         },
+        
+        getFileAndLineInfo: function getFileAndLineInfo() 

```

Comment:
![unittestfwerrormessages](https://user-images.githubusercontent.com/9332678/43107161-9b1354da-8e90-11e8-82ef-24c69b841e85.PNG) Changed the printouts from this file to give the debugger more context into the problem and be less confusing to read. 

Changes: 
- Shows the file and line of the failure. 
- asserts that differ in expectation are now shown in a way that is easier to compare output to expectation